central/debug/service/service.go

```diff
@@ -414,27 +417,47 @@ func getGoroutines(zipWriter *zipWriter) error {
 }
 
 func getLogs(zipWriter *zipWriter) error {
-	if err := getLogFile(zipWriter, "central.log", logging.LoggingPath); err != nil {
-		return err
-	}
-	return nil
+	return getLogFile(zipWriter, "central.log", logging.LoggingPath)
 }
 
-func getLogFile(zipWriter *zipWriter, targetPath string, sourcePath string) error {
-	logFile, err := os.Open(sourcePath)
+func forEachRotation(sourcePath string, f func(filepath string) error) error {
+	dir, fileext := filepath.Split(sourcePath)
+	ext := filepath.Ext(fileext)
+	filename := strings.TrimSuffix(fileext, ext)
+	pattern := filename + "*" + ext
```

Comment:
I'm not sure if we can improve matching so that we are sure files are actually rotations of the provided log file. For example: if we have files `test.log` and `test-module.log` - for `test.log`, information from `test-module.log` will also be appended.

It would be good if we could ensure that we match the format like `-2016-11-04T18-30-00.001`.